index.html

```diff
@@ -162,6 +162,7 @@ <h1>wavesurfer.js examples</h1>
           <li><a href="#multitrack.js">Multi-track</a></li>
           <li><a href="#vowels.js">Vowels</a></li>
           <li><a href="#fm-synth.js">FM synth</a></li>
+          <li><a href="#remounting-container.js">Remounting container</a></li>
```

Comment:
This link might be too long for the sidebar. I’d prefer not to add a whole example for this, since it’s quite an edge case.

Reply:
I mostly used it for testing purposes, removing it.